src/server/database/migrations/20210706024645-modify-checkers-add-isactive-column.js

```diff
@@ -2,19 +2,10 @@
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
-    await queryInterface.addColumn('checkers', 'isActive', {
+    return queryInterface.addColumn('checkers', 'isActive', {
       type: Sequelize.BOOLEAN,
       allowNull: false,
-      defaultValue: false,
-    })
-
-    const query =
-      'UPDATE checkers ' +
-      'SET "isActive" = EXISTS( SELECT 1 FROM checkers as c1, "publishedCheckers" as pc1 ' +
-      'WHERE c1.id = pc1."checkerId" ' +
-      'AND checkers.id = pc1."checkerId")'
-    await queryInterface.sequelize.query(query, {
-      raw: true,
+      defaultValue: true,
```

Comment:
Hmm, wouldn't this result in a situation where checkers that are created are immediately `active`? We only want it to be active on first publish.